qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -472,6 +472,60 @@ def parse_offset_get(s):
         self._buffer2_ready = False
 
         self.connect_message()
+    
+    
+    SNAP_PARAMETERS = {
+            'x': '1',   
+            'y': '2',  
+            'r': '3', 
+            'p': '4',   
+        'phase': '4',  
+           'θ' : '4',
+        'aux 1': '5',  
+        'aux 2': '6',   
+        'aux 3': '7',  
+        'aux 4': '8',  
+         'freq': '9',   
+         'ch 1': '10',
+         'ch 2': '11'  
```

Comment:
as a user, can you confirm that a parameter with a "space" is convenient? I, personally, thought that the most convenient for the users is without the space, like `ch1` instead of `ch 1`.

Reply:
I will not fight ether way for this. without a space is ok too.
My reasoning way that inside a string is not code, so it should follow English prose rules.
But if without spaces is a better style fit, that's fine.